packages/expo-location/src/Location.types.ts

```diff
@@ -457,6 +457,10 @@ export type LocationGeocodedAddress = {
    * @platform ios
    */
   timezone: string | null;
+  /**
+   * TODO: Docs for formattedAddress
+   */
+  formattedAddress: string | null;
```

Comment:
Could you provide more detail about what the `formattedAddress` field contains? Can it be implemented purely on the JS side? Additionally, I wouldn't say that I like this name.

Reply:
`formattedAddress` is like composed string of the address components. Sometimes it can be implemented purely on the JS side depends on the results of `android geocoder` API. The result may not contain all of the address component but `formattedAddress` do.

Example
```
  [
  {
    "adminArea": "Banten",
    "country": "Indonesia",
    "countryCode": "ID",
    "feature": "Wana Kerta",
    "formattedAddress": "",
    "locale": "",
    "locality": "Kecamatan Sindang Jaya",
    "postalCode": "15560",
    "streetName": "",
    "streetNumber": "",
    "subAdminArea": "Kabupaten Tangerang",
    "subLocality": "Wana Kerta"
  }
]
  ```
Since `streetName` and `streetNumber` null, it can't provide full address but, formattedAddress provide it perfectly. I used `formattedAddress` because following [google maps geocoding service API](https://developers.google.com/maps/documentation/javascript/geocoding).

Reply:
It would be great if you could add that information to the comment in the code, replacing the TODO.

Reply:
it's up @lukmccall 5ecb0e